### PR TITLE
Fix hide keyboard when search result

### LINKED ISSUE #291: When open session/speaker detail from search result, Keyboard is not hidden
## Overview (Required)
- title says everything
image

### PR DESCRIPTION
**Pull request by kaya5777**
## Issue
- close #291

## Overview (Required)
- hide keyboard when search result

### CODE REVIEW

**Comment by takahirom** [comment]
Thanks 👍 
If I introduce this, I should implement this for all screen. 😭 
Can you do it in only SearchFragment?

**Comment by kaya5777** [comment]
Sorry, I understand.
I fix only SearchFragment to solve this problem.

File: feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SearchFragment.kt [hunk 1]
```diff
@@ -127,6 +131,17 @@ class SearchFragment : DaggerFragment() {
             searchView.setOnCloseListener { false }
         }
     }
+
+    override fun onPause() {
+        super.onPause()
+
+        val imm = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```
Comment (jmatsu) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SearchFragment.kt]: I'm very sorry, I've forgotten this api requires Android M... so `ContextCompat.getSystemService(requireContext(), InputMethodService::class.java)` looks better. 🙇

File: feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SearchFragment.kt [hunk 2]
```diff
@@ -127,6 +131,17 @@ class SearchFragment : DaggerFragment() {
...
+            context?.systemService<InputMethodManager>()
+        } else {
+            context?.getSystemService(Context.INPUT_METHOD_SERVICE) as? InputMethodManager
+        }
+        imm?.hideSoftInputFromWindow(binding.root.windowToken, 0)
```
Comment (jmatsu) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SearchFragment.kt]: please use the currently focused view instead.

**Pull request merged**